File: app/src/main/java/com/tourtrek/viewModels/TourMarketViewModel.java
package com.tourtrek.viewModels;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import com.tourtrek.data.Tour;

public class TourMarketViewModel extends ViewModel {

    private MutableLiveData<Tour> selectedTour = new MutableLiveData<>();

    public Tour getSelectedTour() {
        return selectedTour.getValue();
    }

    public void setSelectedTour(Tour selectedTour) {
        this.selectedTour.setValue(selectedTour);
    }
}

File: app/src/main/java/com/example/tourtrek/viewmodels/TourMarketViewModel.kt
package com.example.tourtrek.viewmodels

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class TourMarketViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is the Tour Market Fragment"
    }
    val text: LiveData<String> = _text
}